### PR TITLE
Feat/featured-movie-landing-page

### PR DESCRIPTION
**Pull request by mehmetnuricavdar**
# Description Landing page feauterd movie section was created

# Jira task link https://hackyourfuture-dk.atlassian.net/browse/LANDING-25

# Screenshots (if applicable)
Screenshot 2023-05-08 at 02 45 18

### CODE REVIEW

File: packages/client/src/components/FeaturedMovie.js [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+export default function FeaturedMovie() {
+  return (
+    <Card>
+      <Box sx={{ position: 'relative', display: 'flex' }}>
+        <CardMedia
+          component="img"
+          sx={{
+            width: 1672,
+            height: 790,
+            borderRadius: 395,
```
Comment (babak-f) [inline packages/client/src/components/FeaturedMovie.js:21]: How come such a high number for border radius?

File: packages/client/src/components/FeaturedMovie.js [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+            zIndex: 1,
+            top: 238,
```
Comment (babak-f) [inline packages/client/src/components/FeaturedMovie.js:23]: It's better to follow some sort of a system for positions and spacing, so they look consistent to the user. A common practice is to only use multiples of 8.

File: packages/client/src/components/FeaturedMovie.js [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+        >
+          <Typography component="div" sx={{ fontSize: 'h1.fontSize' }}>
+            Ready for Dune?
+          </Typography>
+          <Box sx={{ display: 'flex', alignItems: 'center', mt: 0, pb: 12 }}>
+            <IconButton
+              sx={{ backgroundColor: '#00FFC2' }}
+              aria-label="play/pause"
+            >
+              <PlayArrowIcon sx={{ height: 24.17, width: 24.17 }} />
```
Comment (babak-f) [inline packages/client/src/components/FeaturedMovie.js:53]: Same here - 24.17 seems too specific

**Review by babak-f — COMMENTED** [review]
Good work :)

File: packages/client/src/components/FeaturedMovie.js (hunk 1 above)
Comment (mehmetnuricavdar) [reply packages/client/src/components/FeaturedMovie.js:21]: actually, these dimensions I took from the Figma design template for the FP. For me, they are high but I thought this is the way :D

File: packages/client/src/components/FeaturedMovie.js (hunk 2 above)
Comment (mehmetnuricavdar) [reply packages/client/src/components/FeaturedMovie.js:23]: again because of the Figma all these numbers :D

**Pull request merged**